base/applications/mspaint/winproc.cpp

```diff
@@ -174,17 +159,15 @@ void CMainWindow::InsertSelectionFromHBITMAP(HBITMAP bitmap, HWND window)
 
 LRESULT CMainWindow::OnDropFiles(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled)
 {
-    HDROP drophandle;
     TCHAR droppedfile[MAX_PATH];
-    HBITMAP bmNew = NULL;
-    drophandle = (HDROP)wParam;
-    DragQueryFile(drophandle, 0, droppedfile, SIZEOF(droppedfile));
-    DragFinish(drophandle);
-    LoadDIBFromFile(&bmNew, droppedfile, &fileTime, &fileSize, &fileHPPM, &fileVPPM);
-    if (bmNew != NULL)
-    {
-        UpdateApplicationProperties(bmNew, droppedfile);
-    }
+
+    HDROP hDrop = (HDROP)wParam;
+    DragQueryFile(hDrop, 0, droppedfile, SIZEOF(droppedfile));
+    DragFinish(hDrop);
+
+    if (ConfirmSave())
```

Comment:
Nitpick: You're using && later on. Why not here?

Reply:
Used `&&`.